OrbitLinuxTracing/UprobesUnwindingVisitor.h

```diff
@@ -118,6 +118,8 @@ class UprobesUnwindingVisitor : public PerfEventVisitor {
 
   static std::vector<CallstackFrame> CallstackFramesFromLibunwindstackFrames(
       const std::vector<unwindstack::FrameData>& libunwindstack_frames);
+
+  absl::flat_hash_map<pid_t, std::vector<uint64_t>> uprobe_sp_per_thread_{};
```

Comment:
I would call this one uprobe_sps_per_thread_ as you also used the plural in uprobe_sps.

Reply:
Totally agree!